weblate/templates/mail/account_activity.html

```diff
@@ -29,9 +29,9 @@
 </table>
 
 <p>
-{% blocktrans %}If you don’t recognize this change, you can report it by clicking the button below.{% endblocktrans %}
```

Comment:
This is an optional statement, for what is a nondescript description of an issue.
Why should the user have to recognize changes, and what if the user isn't sure?